docker/intel-dgpu-driver.Dockerfile

```diff
@@ -19,6 +19,14 @@ RUN git clone -b RHEL86_23WW6.5_555_6469.0.3_221221.3 --single-branch https://gi
 RUN git clone -b 23WW06.5_555 --single-branch https://github.com/intel-gpu/intel-gpu-firmware.git && cd intel-gpu-firmware && mkdir -p firmware/license/ && cp -r COPYRIGHT firmware/license/
 
 FROM registry.redhat.io/ubi8/ubi-minimal
+
+LABEL vendor='Intel®'
+LABEL version='devel'
```

Comment:
Please educate me what is version used for? Why it is devel?

Reply:
It is the release version of the image. 'devel' because this is for main branch and not release tag/branch. We need to update this before cutting release. Should we update it with some specific version? any ideas?

Reply:
See my previous comments.

Reply:
According to the link above, how about this for version - 
`version='0.1.0--4.18.0-372.40.1.el8_6.x86_64'`